config/sync/filter.format.os2loop_section.yml

```diff
@@ -0,0 +1,17 @@
+uuid: 3bbfff41-ca40-4b1d-ad01-9fdf51e518a9
+langcode: en
+status: true
+dependencies: {  }
+name: 'Section (rich text)'
+format: os2loop_section
```

Comment:
This should contain `os2loop_section_page`.

Reply:
Fixed